nautobot/core/settings.py

```diff
@@ -430,6 +430,7 @@
 DATE_FORMAT = os.getenv("NAUTOBOT_DATE_FORMAT", "N j, Y")
 DATETIME_FORMAT = os.getenv("NAUTOBOT_DATETIME_FORMAT", "N j, Y g:i a")
 DEBUG = is_truthy(os.getenv("NAUTOBOT_DEBUG", "False"))
+NO_INDEX_ROBOTS = is_truthy(os.getenv("NAUTOBOT_NO_INDEX_ROBOTS", "True"))
```

Comment:
1. This isn't a standard Django setting so this is the wrong "section" of the settings file - this should be somewhere around line 158 instead if keeping the name `NO_INDEX_ROBOTS`
2. `NO_INDEX_ROBOTS` is an unintuitive name to me. How about something like `PUBLISH_ROBOTS_TXT` instead?
3. Need to also add the new setting to `nautobot/core/templates/nautobot_config.py.j2` as well.
4. Are we confident that enabled-by-default is the correct behavior for this new feature?